Source/Adapters/Blackjack.Adapters/UserAdapter.cs

```diff
@@ -39,8 +47,8 @@ public UserAdapter(IUserFacade userFacade)
     public async Task CreateUser(string username)
     {
         string soundex = CreateSoundex(username);
-        UserBE usersBySoundex = await GetUsers(username);
-        if (usersBySoundex != null)
+        UserBE userBySoundex = await GetUsers(username);
```

Comment:
GetUsers needs to be renamed to the singular GetUser

Reply:
we named it plural because everything after the adapter returns a list so you recommended making it Users to reflect that haha. I can change it but there was a reason for it

Reply:
Don't remember that, but if it returns a single user have the name reflect that and ignore whatever I was spoutin previously when I was on crack.

Also while we're at it the fact that we fetch the user by creating a soundex out of their username is an implementation detail that CreateUser doesn't know about and doesn't need to know about. Rename the variable `userBySoundex` to just `userBE` or like `userByUsername`